assets/js/common/CheckCustomizationModal/CheckCustomizationModal.jsx

```diff
@@ -21,20 +23,14 @@ const buildCustomCheckPayload = (checkID, values) => {
   return payload;
 };
 
-const renderLabelWithTooltip = (name) => {
-  const labelContent = (
-    <Label className="block truncate max-w-[200px] sm:max-w-[250px] md:max-w-[300px]">
-      {name}:
-    </Label>
-  );
+const appliedValue = (value) => value?.custom_value ?? value?.current_value;
 
-  return name?.length > 25 ? (
-    <Tooltip zIndex="50" content={name}>
-      {labelContent}
-    </Tooltip>
-  ) : (
-    labelContent
-  );
+const whichInputType = (value) => {
```

Comment:
@nelsonkopliku  in order to make the component more future prove we could use this helper function which allows us to decide which kind of input we want to render.

Reply:
Ok we can start with this. 
Consider also names like `detectType`, `getInputType` etc if they sound better to you.